Statut HSWro.tex

```diff
@@ -41,54 +41,52 @@ \section{CZŁONKOWIE}
         \item zwyczajnych,
         \item honorowych.
       \end{enumerate}
-    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełnia stosowne postanowienia ustawy Prawo o Stowarzyszeniach oraz wszystkie poniższe warunki:
+    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełni wszystkie poniższe warunki:
       \begin{enumerate}
+         \item przedstawi pozytywną opinię co najmniej trzech członków zwyczajnego Stowarzyszenia,
         \item złoży deklarację członkowską na piśmie,
-        \item przedstawi pozytywną opinię co najmniej jednego członka zwyczajnego Stowarzyszenia
+        \item uzyska akceptację deklaracji członkowskiej uchwałą Zarządu Stowarzyszenia.
```

Comment:
Zarządu? Nie powinno być walnego zebrania członków aka sporg?

Reply:
nie. Sporgu juz nie bedzie a jak bedzie to nie bedzie prawny. Chcesz czekac raz do roku jak jest zebranie walne by przyjac jakiegos czlonka?

Reply:
Sporg != walne zebranie